### ISSUE TITLE
How to debug a userscript with requires

### ISSUE THREAD

**Issue by odebroqueville** [violentmonkey/violentmonkey.github.io#56]
This is my userscript, which requires 2 files being server by my local http server:

```
// ==UserScript==
// @name        Readability
// @namespace   Violentmonkey Scripts
// @grant       none
// @require     http://localhost:8000/Readability.js
// @require     http://localhost:8000/Readability-readerable.js
// @version     1.0
// @author      -
// @description 18/01/2024, 21:46:01
// ==/UserScript==

let documentClone = document.cloneNode(true);

/*
    Only instantiate Readability  if we suspect
    the `parse()` method will produce a meaningful result.
*/
if (isProbablyReaderable(documentClone)) {
    let article = new Readability(documentClone).parse();
    document.body.innerHTML = article.content;
}
```

I've included some console logs in the Readability.js file, but they don't make it to my Console!

**Comment by tophf** [comment]
Dependencies are stored locally inside the extension, not loaded directly from the source URL, and are updated periodically according to the auto-update option.

For local development you can use [tracking](https://violentmonkey.github.io/posts/how-to-edit-scripts-with-your-favorite-editor/) from a local file or localhost. In Violentmonkey 2.16 and newer in Chrome you can also [drag-n-drop local files](https://github.com/violentmonkey/violentmonkey/releases/tag/v2.16.0) into the extension.

You can also use a logging breakpoint in devtools instead of adding console.log to the source.

**Comment by odebroqueville** [comment]
@tophf Thank you for your quick response. I've copied my script into a file named readability.user.js created in VS Code. However, I don't understand where I should drag the file to!?!

> Direct 1 for Chrome-like browsers and Violentmonkey 2.16 or newer:
> Drag the file into an open Violentmonkey page or the popup.

What is meant by an open "Violentmonkey page"? I also tried dragging the file onto the popup below, but it just opens a blank page (about:blank%23blocked)!

<img width="321" alt="Screenshot 2024-01-19 at 18 40 02" src="https://github.com/violentmonkey/violentmonkey.github.io/assets/4818315/60fa5a41-5ad1-4f53-af21-c255dc2a3d38">
